WordPress/src/main/res/layout/new_quick_start_task_type_item.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/quick_start_item_root"
+    style="@style/QuickStartTypeRow"
+    android:padding="@dimen/my_site_card_row_padding">
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/quick_start_item_title"
+        style="@style/NewQuickStartTypeTitle"
+        android:layout_width="0dp"
+        android:text="@string/quick_start_sites_type_get_to_know_app"
+        app:layout_constraintEnd_toStartOf="@+id/quick_start_item_icon"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <ImageView
+        android:id="@+id/quick_start_item_icon"
```

Comment:
Np: This illustration image is actually not an icon and can be renamed. Wdyt?

Reply:
👍🏼 I have renamed it to `quick_start_item_img`, let me know if you have any other suggestions. 🤔